src/lib/viewers/doc/DocFindBar.js

```diff
@@ -335,7 +339,8 @@ class DocFindBar extends EventEmitter {
             if (!clicked) {
                 this.findNextButtonEl.focus();
             } else {
-                this.dispatchFindEvent('findagain', false);
+                const type = Browser.isIE() ? 'findagain' : 'again';
```

Comment:
Is this needed? I don't see 'again' listed in the type of events for find?

Reply:
Yeah, it seems that in `pdf_viewer.js` forr 2.84.0, `findagain` is replaced by `again` (line 8907):
```
...
} else if (type === "again") {
      _classPrivateMethodGet(_this, _nextMatch, _nextMatch2).call(_this);
...
```